packages/contracts-bedrock/deploy-config/baklava.json

```diff
@@ -0,0 +1,77 @@
+{
+  "l1StartingBlockTag": "0x382262bec314539f272adb4cdc8165f14019586a87a4301402436c7894eefa47",
+  "l1ChainID": 17000,
+  "l2ChainID": 62320,
+  "l2BlockTime": 1,
+  "l1BlockTime": 12,
+  "maxSequencerDrift": 600,
+  "sequencerWindowSize": 7200,
+  "channelTimeout": 300,
+  "p2pSequencerAddress": "0x4e7361E093De46C2861F8a1A8e7897894a5AB470",
+  "batchInboxAddress": "0xfff0000000000000000000000000000000062320",
+  "batchSenderAddress": "0xE48b8F090e87A563D73Df47a2a5db2Fc749DF530",
+  "l2OutputOracleSubmissionInterval": 600,
+  "l2OutputOracleStartingBlockNumber": 26831470,
+  "l2OutputOracleProposer": "0x2786D8ADFF8690FBC2dd28A53Fab3Bafa5AE6233",
+  "l2OutputOracleChallenger": "0x9e56D6CE53b5fD29528D18bb0002De417977344c",
+  "finalizationPeriodSeconds": 12,
+  "proxyAdminOwner": "0xc07C5A1fBF6c7BC6b4f321E7dd031c0E1E98d32d",
+  "baseFeeVaultRecipient": "0xc07C5A1fBF6c7BC6b4f321E7dd031c0E1E98d32d",
+  "l1FeeVaultRecipient": "0xc07C5A1fBF6c7BC6b4f321E7dd031c0E1E98d32d",
+  "sequencerFeeVaultRecipient": "0xc07C5A1fBF6c7BC6b4f321E7dd031c0E1E98d32d",
+  "finalSystemOwner": "0xc07C5A1fBF6c7BC6b4f321E7dd031c0E1E98d32d",
+  "superchainConfigGuardian": "0xc07C5A1fBF6c7BC6b4f321E7dd031c0E1E98d32d",
+  "baseFeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
+  "l1FeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
+  "sequencerFeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
+  "baseFeeVaultWithdrawalNetwork": 0,
+  "l1FeeVaultWithdrawalNetwork": 0,
+  "sequencerFeeVaultWithdrawalNetwork": 0,
+  "gasPriceOracleOverhead": 0,
+  "gasPriceOracleScalar": 0,
+  "deployCeloContracts": false,
+  "enableGovernance": false,
+  "governanceTokenSymbol": "OP",
+  "governanceTokenName": "Optimism",
+  "governanceTokenOwner": "0xc07C5A1fBF6c7BC6b4f321E7dd031c0E1E98d32d",
```

Comment:
This we don't use these, can we remove them? Not that someone thinks we are subject to OP's governance.

Reply:
We can delete, but we'd need to modify the [config checks](https://github.com/celo-org/optimism/blob/jcortejoso/testnet-configs/op-chain-ops/genesis/config.go#L205).

It may be a change for next dry-run?

Reply:
We'd need to check the code as right now it's required:

```
    │   ├─ [0] VM::parseJsonUint("<stringified JSON>", "$.sequencerFeeVaultWithdrawalNetwork") [staticcall]
    │   │   └─ ← [Return] 0
    │   ├─ [0] VM::parseJsonString("<stringified JSON>", "$.governanceTokenName") [staticcall]
    │   │   └─ ← [Revert] vm.parseJsonString: path "$.governanceTokenName" must return exactly one JSON value
    │   └─ ← [Revert] vm.parseJsonString: path "$.governanceTokenName" must return exactly one JSON value
    └─ ← [Revert] vm.parseJsonString: path "$.governanceTokenName" must return exactly one JSON value
```

Reply:
Ok, then leave that in. We could consider changing it to obvious placeholders, though.

Reply:
If you configure `enableGovernance: false` then it will remove this error

https://github.com/celo-org/optimism/blob/5374bb95c596d198c11eebdfc29dcf8947bb1af5/packages/contracts-bedrock/scripts/deploy/DeployConfig.s.sol#L146